ansible/playbook.yml

```diff
@@ -5,17 +5,17 @@
     - name: Set timezone to Europe/Brussels
       community.general.timezone:
         name: Europe/Brussels
-    - name: Set NTP server to the one from UGent
-      ansible.builtin.lineinfile:
-        path: /etc/systemd/timesyncd.conf
-        regexp: '^#NTP='
-        line: "NTP=ntp.ugent.be"
-    - name: Enable ntp time sync
-      ansible.builtin.systemd:
-        daemon_reload: true
-        name: systemd-time-wait-sync
-        enabled: true
-        state: started
+    # - name: Set NTP server to the one from UGent
```

Comment:
It's still important to have synced time; why was this deleted?

Reply:
My vm kept hanging when using using this, I forgot to uncomment it

Reply:
My vm kept hanging when using this, I forgot to uncomment it